components/auth/actionTypes.js

```diff
@@ -0,0 +1,11 @@
+export const FETCH_USER_REQUEST = 'FETCH_USER_REQUEST';
+export const FETCH_USER_FULFILLED = 'FETCH_USER_FULFILLED';
+export const FETCH_USER_REJECTED = 'FETCH_USER_REJECTED';
+
+export const UPDATE_USER_REQUEST = 'UPDATE_USER_REQUEST';
+export const UPDATE_USER_FULFILLED = 'UPDATE_USER_FULFILLED';
```

Comment:
Perhaps you could consider doing something like this, it is probably less verbose.
```
export const FETCH_USER = ['FETCH_USER_REQUEST', 'FETCH_USER_FULFILLED', 'FETCH_USER_REJECTED']

export const UPDATE_USER = ['UPDATE_USER_REQUEST', 'UPDATE_USER_FULFILLED', 'UPDATE_USER_REJECTED']

export const SAVE = ['SAVE_SESSION_REQUEST', 'SAVE_SESSION']`
```
Then you can destructure the import like so:
```
// import then destructure like below

const [FETCH_USER_REQUEST, FETCH_USER_FULFILLED, FETCH_USER_REJECTED] = FETCH_USER

```

Reply:
Wow, I never thought of this. Thanks! But their order has to be followed if it's an array. What if I use an object?

export const FETCH_USER  = {
       FETCH_USER_REQUEST: 'FETCH_USER_REQUEST',
       FETCH_USER_FULFILLED: 'FETCH_USER_FULFILLED',
}

Then I can do 
const {  FETCH_USER_FULFILLED,  FETCH_USER_REQUEST  } =  FETCH_USER
without being scared of the order

Reply:
go with the object direction, reason...constant look up of keys